webpack.renderer.js

```diff
@@ -40,16 +40,13 @@ const common = {
   },
 };
 
-const renderer = {
-  mode: "production",
-  devtool: "source-map",
-  target: "electron-renderer",
-  node: {
-    __dirname: false,
-  },
-  entry: {
-    "app/main": "./src/app/main.ts",
+const dev = {
+  optimization: {
```

Comment:
Do we need this? During my testing it was sufficient to just set mode.

Reply:
You're right, that was all that was needed. Fixed.